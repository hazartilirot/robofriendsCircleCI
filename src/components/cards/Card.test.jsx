import { shallow } from 'enzyme';
import React from 'react';
import Card from './Card';

it('must render Card Component taking a snapshot of it', () => {
  const wrapper = shallow(<Card />);
  expect(wrapper).toMatchSnapshot();
});