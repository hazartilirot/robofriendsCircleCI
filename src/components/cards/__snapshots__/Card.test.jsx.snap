// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`must render Card Component taking a snapshot of it 1`] = `ShallowWrapper {}`;
